exercises/all-your-base/all_your_base_test.rb

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
```

Comment:
It looks like this may not be needed, since it ran without it in the previous version of the generated test.